docker-compose.yml

```diff
@@ -14,7 +14,7 @@ db:
 #
 web:
   build: .
-  command: /bin/bash -c "sleep 5 && bundle exec rails s -p 4000 -b '0.0.0.0'"
+  command: /bin/bash -c "sleep 5 && bundle exec rails s -p 4000 -b '0.0.0.0' && rake sunspot:solr:start"
```

Comment:
"&&" means 'wait for process to finish and then start this...'. If you want simultaneous execution use only one amperstand. That said, this command is only run when you don't specify a command when you issue `docker run`.